Yosemite/Yosemite/Tools/POS/PointOfSaleOrderService.swift

```diff
@@ -0,0 +1,219 @@
+import Foundation
+import Networking
+
+public struct PointOfSaleCartProduct {
+    let productID: Int64
+    let price: String
+    let productType: ProductType
+
+    public init(productID: Int64, price: String, productType: ProductType) {
+        self.productID = productID
+        self.price = price
+        self.productType = productType
+    }
+}
+
+public struct PointOfSaleCartItem {
```

Comment:
nit: we might want to add a comment to mention how this is different from the `CartItem` in the POS app layer. The POS cart UI might show the cart items differently from how they appear in an order in wp-admin.